core/src/main/java/org/apache/gravitino/Configs.java

```diff
@@ -324,10 +324,11 @@ private Configs() {}
           .intConf()
           .createWithDefault(DEFAULT_METRICS_TIME_SLIDING_WINDOW_SECONDS);
 
-  public static final ConfigEntry<Optional<String>> REST_API_EXTENSION_PACKAGES =
-      new ConfigBuilder("gravitino.extension.restApiPackages")
+  public static final ConfigEntry<Optional<List<String>>> REST_API_EXTENSION_PACKAGES =
```

Comment:
Do we still need `Optional<List<String>>`? I think we can use an empty list as a default value?